extensions.json

```
@@ -158,6 +158,12 @@
       "id": "batisteo.vscode-django",
       "repository": "https://github.com/vscode-django/vscode-django"
     },
+    {
+      "id": "BazelBuild.vscode-bazel",
+      "repository": "https://github.com/bazelbuild/vscode-bazel",
+      "version": "0.4.0",
+      "checkout": "0.4.0"
+    },
```

When I run `node add-extension https://github.com/bazelbuild/vscode-bazel --checkout`, I get this instead:

```suggestion
    {
      "id": "BazelBuild.vscode-bazel",
      "download": "https://github.com/bazelbuild/vscode-bazel/releases/download/0.4.0/vscode-bazel-0.4.0.vsix",
      "version": "0.4.0"
    },
```